### PR TITLE
Inject files into package instead of creating symlinks

### PR DESCRIPTION
**Pull request by kichik**
- symlinks are still not working properly on windows (it fails deleting them because they're actually folders)
- IDE indexes the symlinked folders while they're there which slows things down
- any failure, like a file being in use, causes all symlinks to be left behind

### CODE REVIEW

**Comment by kichik** [comment]
I really have to get my test setup working... That's probably next. It won't even start on Windows :(

**Comment by dschep** [comment]
Yeah, not sure how you'd run the tests on windows. They should be useable in WSL (or anyother way of getting bash on windows). Follow over here to install bats: https://github.com/sstephenson/bats

**Comment by kichik** [comment]
YES!!! It finally works!! 😀 
Thanks for writing such a comprehensive test suite!

**Comment by dschep** [comment]
Awesome! You're welcome, its been reaaaallly useful in making it possible for me to accept PRs with confidence that I'm not breaking anything since we use this heavily at @UnitedIncome (that's the whole reason we wrote it)

I'll try to take a close look at this this weekend since it's a bigger change. With this and the addition of package individually, v4 will be a pretty big release :smiley:

**Comment by dschep** [comment]
Could you update this to resolve the conflicts? Thanks.

**Comment by kichik** [comment]
Is there anything else I can do to help get this merged?

**Comment by dschep** [comment]
Nope, I've just been on vacation thus didn't get around to it :wink:

File: lib/inject.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  const zip = zipper.sync.unzip(packagePath).lowLevel();
+
+  glob.sync([path.join(requirementsPath, '**')], {mark: true, dot: true}).forEach((file) => {
+    if (file.endsWith('/')) {
+      return;
+    }
+
+    const relativeFile = path.relative(requirementsPath, file);
+
+    if (relativeFile.match(/^__pycache__[\\\/]/)) {
```
Comment (dschep) [inline lib/inject.js]: Why omit caches/pyc? IIRC they're cross platform(just tied to the version of python, which should be the same) and should provide a small perf boost.

Comment (kichik) [reply lib/inject.js]: It's just the root `__pycache__` which contains mostly `six.pyc` that's not deployed by default and maybe other small files. This mimics the old behavior https://github.com/UnitedIncome/serverless-python-requirements/pull/147/files#diff-0ffe4d151b22eb4ec361c9b4faaa9b11L50 and might be improved in the future. 

I actually plan on trying to get rid of all `.pyc` files in the future with the `--no-compile` flag to provide more stable builds (as in building twice produces the exact same result). But there's still some more research to be done there.

**Pull request merged**